topic04-git-introduction/book-git-tutorial/02.Branching.md

```diff
@@ -0,0 +1,34 @@
+# Branching With GitHub
+
+----
+## What is Branching?
+
+
+This document is to show how branching works in git and how to use it. Branching is a feature available in git that lets you pull off from the master branch to make changes that won't directly effect the master branch unless merged. It also means that others have to review your work first just in case you have errors in your work such as spelling or bugs. 
+
+
+## Creating a Branch.
```

Comment:
Would perhaps add a section 'why you might want to use branching'. Also, would like to see the 'Creating a Branch' section expanded out and sub-headings/ bullet points to give more detail on each step